JASP-Engine/JASP/R/common.R

```diff
@@ -2181,9 +2181,10 @@ as.list.footnotes <- function(footnotes) {
 		eval(plot())
 		if (obj) plot <- recordPlot() # save plot to R object
 	} else if (isRecordedPlot) { # function was called from editImage to resize the plot
-	    .redrawPlot(plot) #(see below)
+	  .redrawPlot(plot) #(see below)
 	} else {
-		print(plot)
+		# plot objects such as ggplot and qgraph
+		plot(plot)
```

Comment:
The syntax is a bit confusing. In addition, I add a flag, just to display that this doesn't need to be accepted for 0.9